### PR TITLE
Introducing Flow Panel plugin

### PR DESCRIPTION
**Pull request by hellt**

### CODE REVIEW

File: configs/gnmic/gnmic-config.yml [hunk 1]
```diff
@@ -3,127 +3,116 @@
...
-port: 57400
 timeout: 10s
-skip-verify: true
-encoding: json_ietf
+
+common_srl_subscriptions: &common_srl_subs
+  port: 57400
+  skip-verify: true
+  insecure: false
+  password: NokiaSrl1!
```
Comment (hellt) [inline configs/gnmic/gnmic-config.yml:12]: I think these could stay at the top of the config, as they are connection details

Comment (FloSch62) [reply configs/gnmic/gnmic-config.yml:12]: I prefer it like this, with that its easier to add sros if needed. For example:

`username: admin
password: *******
insecure: true
log: true

common_sros_subscriptions: &common_sros_subs
  insecure: true
  password: ******
  subscriptions:
      - sros-if-stats
      - sros-bgp-stats
      - sros-service-stats

common_srl_subscriptions: &common_srl_subs
  port: 57400
  skip-verify: true
  insecure: false
  password: *******
  subscriptions:
      - srl-if-stats 
      - srl-bgp
      - srl-system-performance
      - srl-routes
      - srl-bridge
      - srl-apps
      - srl-net-instance`

**Comment by FloSch62** [comment]
I should update all the old screenshots and gifs of the old plugin, I guess...

**Comment by hellt** [comment]
great work @FloSch62 
did a few styling edits, but the rest looks amazing
mergin

**Pull request merged**